tests/cephfs/cephfs_scale/cleanup.py

```diff
@@ -0,0 +1,131 @@
+import json
+import time
```

Comment:
Are you planning to the tests to suite file in this PR ?

Reply:
Yes, I have added suite file too.